### ISSUE TITLE
print a large recyclerview

### ISSUE THREAD

**Issue by FBertanha** [natario1/ViewPrinter#17]
how break pages to print all itens of a recyclerview

**Comment by anggit97** [comment]
this issue not solve yet? i have same problem for this issue too. please i need help...

**Comment by FBertanha** [comment]
not yet, actually i'm using itext pdf for print

**Comment by anggit97** [comment]
It is external library?

**Comment by FBertanha** [comment]
yes, its a external library, but you need to create the elements manually. [Exploring iText : To Create PDF in Android](https://medium.com/android-school/exploring-itext-to-create-pdf-in-android-5577881998c8)

**Comment by anggit97** [comment]
Owh I see, I have looked at the link you gave. How about if I have a chart in my view? Can it inflate in itext?

**Comment by jonglee01** [comment]
> yes, its a external library, but you need to create the elements manually. [Exploring iText : To Create PDF in](https://medium.com/android-school/exploring-itext-to-create-pdf-in-android-5577881998c8) [Android](https://apkbeach.com/dragon-city-mod-apk/)
